frontends/java/oss-fuzz-main.py

```diff
@@ -56,25 +57,46 @@ def find_fuzz_targets(path):
   print("Finding fuzz targets in %s" % path)
   jar_files = set()
   targets = list()
-  for classfile in os.listdir(path):
+  for wrapper_script in os.listdir(path):
     print("Checking target %s" % classfile)
-    if not classfile.endswith(".class"):
-      continue
-
     # Check if wrapper script exists and whether it has the right tag.
-    wrapper_script = classfile.replace(".class", "")
     if not os.path.isfile(wrapper_script):
       continue
     with open(wrapper_script) as wrapper_script_fd:
       if 'LLVMFuzzerTestOneInput' not in wrapper_script_fd.read():
         continue
 
+    classfile = "%s.class" % wrapper_script
+    jar_file = "%s.jar" % wrapper_script
+
     # Create relevant .jar file if it is not there.
-    jar_file = classfile.replace(".class", ".jar")
     if not os.path.exists(jar_file):
       subprocess.check_call("jar cvf %s %s" % (jar_file, classfile), shell=True)
 
+    # Extract jar file for package name discovery.
+    temp_dir = os.path.join(path, 'temp-jar')
+    os.mkdir(temp_dir)
+    subprocess.check_call("cd %s && jar xvf %s/%s" % (temp_dir, path, jar_file), shell=True)
```

Comment:
Please use `cwd` from https://docs.python.org/3/library/subprocess.html#subprocess.check_call instead of `cd && ..`

Reply:
done